src/ckb-daemon/usb.h

```diff
@@ -39,6 +39,7 @@
 #define P_K95_PLATINUM  0x1b2d
 #define P_K95_PLATINUM_STR "1b2d"
 #define IS_K95(kb)      ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95 || (kb)->product == P_K95_NRGB || (kb)->product == P_K95_PLATINUM))
+#define IS_K95P(kb)     ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95_PLATINUM))
```

Comment:
If there's a test specifically for K95 Platinum now, is there a need to keep testing for it in IS_K95 macro?